packages/docs/page-config/styles/colors/components/components/PaletteGridCard.vue

```diff
@@ -7,6 +7,8 @@
       <p>
         {{ name }}
       </p>
+      <!-- Is there any way to specify font-size in px in tailwind? I found text-sm
+      change font-size in rem. change it to text-sm cause this p element to move.-->
       <p style="font-size: 14px;">
```

Comment:
`text-sm` changes both `font-size` and `line-height`, in the unit of `rem`. Tailwind can be configured but I don't know this is a good way, and even it is, I haven't found any `tailwind.config.js` exist under docs. Is there any way to solve this?